Hashlink.js

```diff
@@ -23,8 +24,8 @@ export class Cipher {
    * is provided, the hashlink is created from the data.
    *
    * @param {Object} options - The options for the create operation.
-   * @param {Array} options.data - The data associated with the given URL. If
-   *   provided, this data is used to create the cryptographic hash.
+   * @param {Uint8Array} options.data - The data associated with the given URL.
```

Comment:
I think this is how you do optional params:

```suggestion
   * @param {Uint8Array} [options.data] - The data associated with the given URL.
```